airbyte-integrations/connectors/source-google-sheets/unit_tests/test_helpers.py

```diff
@@ -227,6 +227,31 @@ def mock_client_call(spreadsheetId, includeGridData, ranges=None):
 
         self.assertEqual(expected, actual)
 
+    def test_get_spreadsheet_id(self):
+        test_url = "https://docs.google.com/spreadsheets/d/18vWlVH8BfjGegwY_GdV1B_cPP9re66xI8uJK25dtY9Q/edit#gid=1820065035"
```

Comment:
This should be parametrized using pytest's parametrization, but it's not a particularly high leverage use of time for you right now. 
```suggestion
        # TODO: make this parametrized
        test_url = "https://docs.google.com/spreadsheets/d/18vWlVH8BfjGegwY_GdV1B_cPP9re66xI8uJK25dtY9Q/edit#gid=1820065035"
```